GOVERNANCE.md

```diff
@@ -0,0 +1,38 @@
+# in-toto governance
+This document covers the project's governance and committer process.  The
+project consists of the in-toto
+[specification](https://github.com/in-toto/docs) and
+[reference implementation](https://github.com/in-toto/in-toto).
+
+## Maintainership and Consensus Builder
+The project is maintained by the people indicated in
+[MAINTAINERS.txt](MAINTAINERS.txt).  A maintainer is expected to (1) submit and
+review GitHub pull requests and (2) open issues or [submit vulnerability
+reports](https://github.com/in-toto/in-toto#security-issues-and-bugs).
+A maintainer has the authority to approve or reject pull requests submitted by
+contributors.  The project's Consensus Builder (CB) is
+Justin Cappos <jcappos@nyu.edu, @JustinCappos>.
+
+## Contributions
+A contributor can [submit GitHub pull
+requests](https://github.com/in-toto/in-toto#instructions-for-contributors)
+to the project's repositories.  They must follow the project's [code of
+conduct](CODE-OF-CONDUCT.md), the [code style
+guidelines](https://github.com/secure-systems-lab/code-style-guidelines), and
+must unit test any new software feature or change.  Submitted pull requests
+undergo review and automated testing, including, but not limited to:
+
+* Unit and build testing via [Travis CI](https://travis-ci.org/) and
+[Tox](https://tox.readthedocs.io/en/latest/).
+* Static code analysis via [Pylint](https://www.pylint.org/) and
+[Bandit](https://wiki.openstack.org/wiki/Security/Projects/Bandit).
+* Review by one or more maintainers
```

Comment:
nitpick: You should consider ending this item with a period, since the other listed items all end in a period.